src/main/resources/pdf_templates/ds01.html

```diff
@@ -0,0 +1,229 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en">
+
+<head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type"/>
+    <title>DS01</title>
+
+    <style type="text/css">
+        body {
+            padding: 20px 40px 0 40px;
+            font-size: 14px;
+            font-family: 'TransportNewLight-Regular', sans-serif;
+        }
+
+        img {
+            height: 100%;
+            width: 100%;
+        }
+
+        hr {
+            border: 0.7px solid black;
+        }
+
+        .ch-logo {
+            margin-top: 20px;
+            width: 280px;
+            height: 40px;
+            float: left;
+        }
+
+        .application-type {
+            float: right;
+            text-align: right;
+            margin-right: 10px;
+            font-weight: bold;
+            width: 300px;
+        }
+
+        .application-type-big {
+            font-size: 55px;
+        }
+
+        .application-type-small {
+            font-size: 18px;
+        }
+
+        header {
+            display: block;
+            height: 270px;
+        }
+
+        h1 {
+            width: 350px;
+            float: right;
+            border: 5px solid black;
+            padding: 10px 15px 5px 15px;
+            text-align: center;
+            font-size: 30px;
+            display: block;
+            font-weight: normal;
+            margin-top: 0px;
+            margin-bottom: 5px;
+        }
+
+        h2 {
+            text-align: center;
+            font-weight: bold;
+            font-size: 20px;
+            margin-top: 10px;
+            margin-bottom: 10px;
+        }
+
+        h3 {
+            font-weight: bold;
+            margin-bottom: 5px;
+            margin-top: 10px;
+        }
+
+        p {
+            margin-top: 15px;
+            margin-bottom: 20px;
+            line-height: 20px;
+        }
+
+        .company-data {
+            margin-top: 20px;
+            width: 100%;
+        }
+
+        .company-data table tr td:first-child {
+            width: 175px;
+        }
+
+        .company-data table tr {
+            height: 25px;
+        }
+
+        .director-data-table tr td:first-child {
+            padding-left: 10px;
+            width: 210px;
+        }
+
+        .director-data-table tr td:nth-child(2) {
+            width: 320px;
+            padding-top: 30px;
+        }
```

Comment:
yeah let's revert this back to how it was before. Once you start using nth-child syntax it starts getting difficult to maintain in future. Just add a class to the td's to differentiate them like you had before.